book.rb

```diff
@@ -0,0 +1,17 @@
+require './rentals'
+
+class Bo
+  attr_reader :rentals
```

Comment:
Accordance to (A book has many rentals),
- [ ] Setters for @rentals should be present.